src/backend/app.py

```diff
@@ -98,12 +98,18 @@ def mainpage_get(mainpage_obj: MainPage, args: MultiDict):
         apts = []
         price_sort = 0
         rating_sort = 0
+        apt_id = -1
         if param.price_sort is not None:
             price_sort = param.price_sort
 
         if param.rating_sort is not None:
             rating_sort = param.rating_sort
-        apts = mainpage_obj.populate_apartments(param.num_apts, price_sort, rating_sort)
+
+        if param.apt_id is not None:
+            apt_id = param.apt_id

```

Comment:
`What is the point of this variable?`

Reply:
It allows support for pagination as `apt_id` is the only unique property of an apartment in the DB.